service/src/main/java/uk/gov/hmcts/reform/fpl/service/FurtherEvidenceNotificationService.java

```diff
@@ -21,14 +21,35 @@
 @Service
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class FurtherEvidenceNotificationService {
-    private final InboxLookupService inboxLookupService;
+    private final LocalAuthorityRecipientsService localAuthorityRecipients;
     private final NotificationService notificationService;
 
     private final FurtherEvidenceUploadedEmailContentProvider furtherEvidenceUploadedEmailContentProvider;
 
-    public Set<String> getLocalAuthoritySolicitorEmails(CaseData caseData) {
-        return inboxLookupService.getRecipients(
-            LocalAuthorityInboxRecipientsRequest.builder().caseData(caseData).build());
+    public Set<String> getLocalAuthoritiesRecipients(CaseData caseData) {
```

Comment:
This change might require some small discussion with @meganaslattery, I know that she was updating code in a similar area